test/evergreen.yml

```diff
@@ -2537,6 +2537,8 @@ tasks:
           script: |
             set -o errexit
             set -o verbose
+            sudo su
+            ulimit -n 1000000
```

Comment:
I believe it's an anti-pattern to update system-level settings in an Evergreen task. Did you check with the Build team to see whether they could write the required ulimit value into the distro image? Our team should be the only user for the `ubuntu1804-wt-large` distro.

Reply:
I asked them the question if it was possible, will keep you updated.

Reply:
See https://jira.mongodb.org/browse/BUILD-13585

Reply:
As discussed, we will leave this for now and remove it as part of https://jira.mongodb.org/browse/WT-7842